packages/keymaps/README.md

```diff
@@ -0,0 +1,21 @@
+# Theia - Keymaps Extension
+
+See [here](https://github.com/theia-ide/theia) for a detailed documentation.
+
+An extension that allows the user to reconfigure default keybindings with custom keymaps. By modifying the appropriate `keymaps.json`, the user can modify existing keybindings, or add keybindings to commands that do not yet have a keybinding associated to them.
+
+Example of a valid `keymaps.json` file
+
+```json
+[
+    {
+        "command": "quickCommand",
+        "keybinding": "ctrl+shift+f4"
+    }
+]
```

Comment:
I think we need to define the complete list of 'easy keys' here.

Reply:
Would a link to `keys.ts` with the correct line number of the map be enough as a reference for the readme?

Reply:
I think since these are static and may be geared at end users it would be better to copy them in the readme.

Reply:
See how they specific this here: https://craig.is/killing/mice as example?

Reply:
With that example that makes sense, I'll add it to the readme.